keyboards/mwstudio/alicekk/config.h

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2022 TW59420 <https://github.com/TW59420>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x7BA1
+#define PRODUCT_ID      0x3201
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    MWStudio
+#define PRODUCT         AlickKK
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { A3, A5, A6, A7, B0 }
+#define MATRIX_COL_PINS { A10, A9, A8, B15, B14, B13, B12, A4, A2, A1, B6, B5, B4, B3, A15 }
+
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION ROW2COL
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+#define FORCE_NKRO
+
+/* 将USB 轮询速率更改为 1000hz 并为精英游戏每次扫描使用更大的密钥*/
+#define USB_POLLING_INTERVAL_MS  2
+#define QMK_KEYS_PER_SCAN  12
```

Comment:
polling rate is set to 1 by default, actually.  And the keys per scan probably shouldn't be used
```suggestion
```